src/containers/FormikForm/AddNotesField.tsx

```diff
@@ -96,15 +104,18 @@ const AddNotesField = ({
               data-testid="notesInput"
```

Comment:
Placeholder blir aldri sendt ned, så den vil ikke finnes, stemmer for at du bare setter den direkte her!